openmdao/test/record_util.py

```diff
@@ -0,0 +1,33 @@
+import inspect
```

Comment:
I looked at the coveralls report and that file does not even show up which I guess is a good sign that it is not used at all!